Storage/SQL/SQLiteHistory.swift

```diff
@@ -541,6 +559,34 @@ extension SQLiteHistory: BrowserHistory {
             "LIMIT \(historyLimit)",
         ].joined(separator: " ")
 
+        if usePregeneratedTempTable && AppConstants.BuildChannel != .release {
+            db.withConnection { connection -> Void in
+                do {
+                    try connection.executeChange("select * FROM sqlite_master WHERE name='\(AwesomebarBookmarksTempTable)'")
+                } catch {
+                    assertionFailure("Fatal: usePregeneratedTempTable is true without having called createTempTablesForFrecencyQuery()")
+                }
+            }
+        }
+
+        let bookmarksSQL = [
+            "SELECT NULL AS historyID, url, title, guid, NULL AS domain_id, NULL AS domain,",
+            "visitDate AS localVisitDate, 0 AS remoteVisitDate, 0 AS localVisitCount,",
+            (includeIcon ? "iconID, iconURL, iconDate, iconType, iconWidth," : ""),
+            "0 AS remoteVisitCount,",
+            "visitDate AS frecencies,",  // Fake this for ordering purposes.
+            "0 as maxFrecency,", // Need this column for UNION
+            "1 AS is_bookmarked",
+            "FROM \(usePregeneratedTempTable ? AwesomebarBookmarksTempTable : ViewAwesomebarBookmarks)",
+            whereClause,                  // The columns match, so we can reuse this.
+            "GROUP BY url",
+            "ORDER BY visitDate DESC LIMIT \(bookmarksLimit)",
+        ].joined(separator: " ")
+
+        // @TODO remove includeIcon case entirely. It adds code complexity and is slowing down queries when the favicon isn't being displayed
```

Comment:
If it's not being used, then presumably flipping the default to `false` is an easy immediate step?

Reply:
Good point, I'll try that.